test-data/unit/check-type-aliases.test

```diff
@@ -1238,3 +1238,15 @@ from typing import List, Union
 A = Union[int, List[A]]
 def func(x: A) -> int: ...
 [builtins fixtures/tuple.pyi]
+
+[case testAliasExplicitNoArgsNotGeneric]
+from typing import List, assert_type
+from typing_extensions import TypeAlias
+
+Implicit = List
+Explicit: TypeAlias = List
```

Comment:
Also test with non-alias generic class (e.g. `<alias> = list`)? You could also test with `tuple`, since it's a bit special.